packages/chrome/stories/examples/Default.tsx

```diff
@@ -81,9 +87,11 @@ export const Default: Story<IDefaultStoryProps> = ({
   isExpanded,
   showSubnav,
   showSidebar,
+  showSheet,
   hueColor,
   itemCount
 }) => {
+  const [_, updateArgs] = useArgs();
```

Comment:
What does `_` mean? Im confused.

Reply:
It doesn't have a meaning really, it was meant to be a placeholder and treating the array like parameters (which isn't necessary). Will update to `[, updateArgs]`